scripts/docker_image/Dockerfile

```diff
@@ -4,4 +4,8 @@ ARG BASE_DIR
 
 COPY ${BASE_DIR}/dxCompiler-$VERSION.jar /dxCompiler.jar
 
-ENTRYPOINT ["java", "-jar", "/dxCompiler.jar"]
```

Comment:
Only add the warning script to the legacy Docker image (that's published as part of the release GHA). The other Dockerfile (I think it's this one in scripts/docker_image) is part of the instructions to DIY it, which is the new way.

Reply:
I see, I just used it for tests.

Reply:
fixed in [439e46e](https://github.com/dnanexus/dxCompiler/pull/481/commits/439e46e14198b41b361e2ed325662acef845179c)